lib/scrabble.rb

```diff
@@ -1,7 +1,48 @@
 class Scrabble
 
+  def split_word(word)
+    word.chars
+  end
+
+  def capitalize_letters(letters)
```

Comment:
This method & the next are great use-cases for map & show a strong effort to break out logic!